manager.go

```diff
@@ -249,3 +272,49 @@ func (m *Manager) GetRetries(page uint64, pageSize int64, match string) (Retries
 		RetryJobs:       retryJobs,
 	}, nil
 }
+
+func (m *Manager) startHeartbeat() error {
+	if !m.opts.Heartbeat {
+		return nil
+	}
+
+	m.sendHeartbeat()
+
+	heartbeatTicker := time.NewTicker(5 * time.Second)
+	m.heartbeatChannel = make(chan bool, 1)
+
+	go func() {
+		for {
+			select {
+			case <-heartbeatTicker.C:
+				m.sendHeartbeat()
+			case <-m.heartbeatChannel:
+				return
+			}
+		}
+	}()
+	return nil
+}
+
+func (m *Manager) removeHeartbeat() error {
+	if !m.opts.Heartbeat {
```

Comment:
It looks like the calling code does this options check and so this may not really be needed?  Same with `sendHeartBeat()`.  Maybe this was a case of "better safe than sorry?"